.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "main"
+      - "master"
+  pull_request:
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  unit-test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-18.04
```

Comment:
We don't need to add extra test for ubuntu 18?

Reply:
Done